### ISSUE TITLE
Feature: Support Multi-label classification outputs

### ISSUE THREAD

**Issue by eddiebergman** [mljar/mljar-supervised#311]
# Feature
Having the library support Multi-label/mult-target outputs would be nice. As you mentioned,  `MultiOutputClassifier` from `sklearn` could be used to adapt the library in a general case.

# Benefit
Increased usability in different domains and datasets.

**Comment by pplonski** [comment]
@eddiebergman thank you for the feature request! Do you have example datasets that will be good for testing / benchmarking this functionality?

**Comment by eddiebergman** [comment]
Unfortunately, my use case is currently on non-public modified data.

For testing,  `sklearn.datasets.make_multilabel_classification` [doc](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.make_multilabel_classification.html) could be used. 

As for a benchmark I could point you to the follow datasets from [OpenML](https://www.openml.org),
they are all tabular datasets:
* [yeast](https://www.openml.org/d/41473) - 14 classes, genetic
* [emotions](https://www.openml.org/d/41545) - 6 classes, feature extracted audio
* [image](https://www.openml.org/d/41546) - 5 classes, feature extracted images
* [reuters](https://www.openml.org/d/41547) - 7 classes, feature extracted text

**Comment by pplonski** [comment]
Great, thank you @eddiebergman :+1:

**Comment by AkshayNovacene** [comment]
Hello @pplonski I wanted to know if MlJar still doesn't support Multi-label classification?

**Comment by pplonski** [comment]
Hi @AkshayNovacene,

We dont support multi label classification. We will talk onternally with @Bocianski and maybe we will add it soon, we will let you know.
